pkg/urbit/jets/e/pedersen.c

```diff
@@ -0,0 +1,100 @@
+
+/* j/5/pedersen.c
+**
+*/
+#include "all.h"
+#include <urcrypt.h>
+
+static c3_y*
+u3r_bytes_all_works(c3_w* len_w, u3_atom a)
```

Comment:
Style issue: rename this something meaningful, and make it a local name. Something starting with an underscore. Also, why is this function here? Please explain with a comment.